composer.json

```
@@ -15,17 +15,23 @@
     "require": {
         "php": "^8.0",
         "ext-bcmath": "*",
+        "ext-ctype": "*",
         "ext-exif": "*",
 		"ext-fileinfo": "*",
```

```suggestion
        "ext-fileinfo": "*",
```

Replacing two tabs by eight spaces.